apps/employee-monitoring/src/components/modal/settings/system-logs/ViewSystemLogModal.tsx

```diff
@@ -20,61 +25,137 @@ const ViewSystemLogModal: FunctionComponent<ViewSystemLogModalProps> = ({
   closeModalAction,
   setModalState,
 }) => {
+  // useSWR
+  const {
+    data: swrSystemLogDetails,
+    isLoading: swrIsLoading,
+    error: swrError,
+  } = useSWR(modalState && rowData.id ? `/user-logs/${rowData.id}` : null, fetcherEMS, {
+    shouldRetryOnError: false,
+    revalidateOnFocus: false,
+  });
+  // Zustand initialization
+  const {
+    SystemLogDetails,
+    SetGetSystemLogDetails,
+
+    ErrorSystemLog,
+    SetErrorSystemLog,
+  } = useSystemLogsStore((state) => ({
+    SystemLogDetails: state.getSystemLogDetails,
+    SetGetSystemLogDetails: state.setGetSystemLogDetails,
+
+    ErrorSystemLog: state.errorSystemLog,
+    SetErrorSystemLog: state.setErrorSystemLog,
+  }));
+
+  // isLoading
+  useEffect(() => {
```

Comment:
not sure what is the aim for setting back to empty and already default empty object. Check the logic again